README.md

```diff
@@ -1,15 +1,77 @@
-# Solana node helpers
+# Solana helpers
 
-The `node-helpers` package package contains node.js specific Solana helper functions.
+The `helpers` package contains Solana helper functions, for use in the browser and/or node.js
 
-See `@solana/web3.js` for functions that work in both the browser and node.js.
+Eventually most of these will end up in `@solana/web3.js`.
 
 ## Installation
 
 ```bash
 npm i @solana-developers/node-helpers
 ```
 
+# helpers for the browser and node.js
+
+## getCustomeErrorMessage()
```

Comment:
this should probably be getCustomErrorMessage

Reply:
Good spot, fixed.